UDPServer/IServerInterface.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UDPServer
+{
+    interface IServerInterface
+    {
+        string ConnectionIP { get; set; }
+
+        int ConnectionPort { get; set; }
+
+        string ReceivedData { get; set; }
+
+        void Connect();
+
+        void Disconnect();
+
+        //Task GetData();
```

Comment:
If this is not used, remove it. Always make sure that commented out code is removed before committing.

Reply:
Removed.